apps/chatbot/docker/compose.test.yaml

```diff
@@ -10,7 +10,7 @@ services:
     volumes:
       - ..:/app
       - ./files/.aws:/root/.aws
-      - ./files/nextjs-website/out:/app/build-devp/out
+      - ../../nextjs-website/out:/app/build-devp/out
```

Comment:
`./files/nextjs-website/out:/app/build-devp/out` is correct, because I want only a subset of generated documents, overwise the test will be longer

Reply:
done